phalcon/Mvc/Collection/ManagerInterface.zep

```diff
@@ -41,56 +40,56 @@ interface ManagerInterface
     /**
      * Binds a behavior to a collection
      */
-    public function addBehavior(<CollectionInterface> model, <BehaviorInterface> behavior);
+    public function addBehavior(<CollectionInterface> collection, <BehaviorInterface> behavior);
 
     /**
-     * Returns the connection related to a model
+     * Returns the connection related to a collection
      */
-    public function getConnection(<CollectionInterface> model) -> <AdapterInterface>;
+    public function getConnection(<CollectionInterface> collection);
```

Comment:
what about return type?

Reply:
getConnection returns a "MongoDB Database", do I have to correct that? In phalcon:3.4.4.x, getConnection returned "\MongoDB"

Reply:
as a far as I see it should return an interface/abstract class instance, isn't?
```zep
        // ...
        let connection = container->getShared(service);
        // ...

        return connection;
```

Reply:
I didn't change this part of the code, it returns an instance of \MongoDB\Database;  in 3.4.x it returned an instance of \Mongo

Reply:
But now it isn't 3.4. Let's change method signature